src/Pages/ProductInfo/ProductDetail.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import DetailInfo from './DetailInfo';
+import Reviews from './Reviews';
+import './ProductDetail.scss';
+
+class ProductDetail extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      activeID: 0,
+      isClicked: 'select-btn',
+    };
+  }
+
+  clickHandleTab = e => {
+    const id = parseInt(e.target.id);
+    this.setState({ activeID: id, isClicked: 'select-btn' });
+  };
+
+  render() {
+    const tab = {
+      0: <DetailInfo />,
+      1: <Reviews />,
+    };
+    return (
+      <div className="product-detail-conponent">
+        <div className="detail-tab">
+          <div>
+            <button
+              name="info"
+              id="0"
+              className={!this.state.activeID ? `${this.state.isClicked}` : ''}
```

Comment:
- 근데 여기에서는 사실, isClicked 라는 스테이트가 아예 필요 없어 보이기도 하네요!
```suggestion
              className={!this.state.activeID ? 'select-btn' : ''}
```

Reply:
오... 이렇게 바꾸면 렌더링을 한번 더 줄일 수 있다고 생각해도 괜찮을까요?
수정 완료 했습니다!